docs/en/observability/monitor-kubernetes.asciidoc

```diff
@@ -0,0 +1,223 @@
+[[monitor-kubernetes]]
+= Monitor Kubernetes: Observe the health and performance of your Kubernetes deployments
+
+*Open issues:* 
+
+* What assumptions can we make about the user wrt Kubernets knowledge? Can we
+assume that users are familiar with how Kubernetes works and the overall
+architecture?
+* Do we want to discuss prometheus monitoring?
+* Do we want to cover APM, Heartbeat, and Packetbeat in this tutorial? (current
+outline assumes that we do)
+
+[discrete]
+== What you’ll learn
+
+In this guide, you’ll learn how to use the {stack} to collect, analyze, and
+correlate logs and metrics from containers, applications, and services
+running on top of Kubernetes.
+
+This guide describes how to deploy Elastic monitoring agents as DaemonSets.
+For other deployment options, see:
+
+* {eck-ref}/index.html[{ecloud} on Kubernetes (ECK)]
+* https://github.com/elastic/helm-charts/blob/master/README.md[Helm charts]
+
+[discrete]
+== Before you begin
+
+TODO: List the prerequisites here. For example, {es} for storing and searching
```

Comment:
Do we have a preferred way for this, like Elastic Cloud?

Reply:
Yes, we have some standard cloud-first language that we'll use to describe the stack prerequisites. What I need help with are other prereqs, for example, set up that's required in K8s or whatever else users need to set up or have access to before they begin.

Reply:
👍